icecreamshop/src/test/java/com/summerschool/icecreamshop/controller/RateControllerTest.java

```diff
@@ -49,6 +51,17 @@ public void setUp(){
         rateDTO.setRate(4);
     }
 
+    @Test
+    public void testCreateRate() {
+
+        Mockito.when(modelMapper.map(rateDTO, Rate.class)).thenReturn(rate);
+        Mockito.when(rateService.add(rate)).thenReturn(rate);
+
+        ResponseEntity<RateDTO> r = rateController.add(rateDTO);
```

Comment:
Schimba numele lui r. De regula variabele trebuie sa aiba un nume care sa le arate si tipul, de exemplu in cazul asta responseEntity.